index.js

```diff
@@ -314,11 +313,7 @@ RedisClient.prototype.init_parser = function () {
 
     // "reply error" is an error sent back by Redis
     this.reply_parser.on("reply error", function (reply) {
-        if (reply instanceof Error) {
-            self.return_error(reply);
-        } else {
-            self.return_error(new Error(reply));
-        }
+        self.return_error(reply);
```

Comment:
Is this safe? I feel like there were cases that the error was a non-Error and needed to be wrapped.